Entity/Repository/FaqRepository.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Grossum\FaqBundle\Entity\Repository;
+
+use Doctrine\ORM\EntityRepository;
+use Grossum\FaqBundle\Entity\Faq;
+
+class FaqRepository extends EntityRepository
+{
+    /**
+     * @return Faq[]
+     */
+    public function findAllEnabled()
+    {
+        return $this
```

Comment:
@cowboy4ik89 you can use `return $this->findBy(['enabled' => true])`. That's all